### PR TITLE
Feature/events load

### PR DESCRIPTION
**Pull request by adindotdev**
Only merged after #13 is approved and merged.

Pulls events from the backend. Saves it to device storage if it worked, loads from device storage if it didn't.

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
```
Comment (jindazhao) [inline ios/Podfile:1]: I don't know what any of this file does

File: ios/Podfile.lock [hunk 2]
```diff
@@ -0,0 +1,137 @@
+PODS:
```
Comment (jindazhao) [inline ios/Podfile.lock:1]: Also please explain what this file does too

File: ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -7,13 +7,15 @@
 	objects = {
```
Comment (jindazhao) [inline ios/Runner.xcodeproj/project.pbxproj:7]: I'm assuming all file up to here are for ios development?

File: ios/Podfile (hunk 1 above)
Comment (adindotdev) [reply ios/Podfile:1]: Generated files necessary for setting up the libraries to work on the iOS side

File: ios/Podfile.lock (hunk 2 above)
Comment (adindotdev) [reply ios/Podfile.lock:1]: Lock files are generally generated files that list dependency versions (all our libraries listed with their versions)

File: ios/Runner.xcodeproj/project.pbxproj (hunk 3 above)
Comment (adindotdev) [reply ios/Runner.xcodeproj/project.pbxproj:7]: Yes, I cannot explain what this one does tho LOL

**Review by jindazhao — APPROVED** [review]

**Pull request merged**